documentation/onboard-dpg-in-sdkautomation/java/README.md

```diff
@@ -1,45 +1,47 @@
-# Add Autorest Configuration of Java SDK 
+# Add TypeSpec Configuration for Java SDK
 
-## Parameter Description
+See [Use SDK Automation from REST API specifications](https://github.com/Azure/azure-sdk-for-java/wiki/TypeSpec-Java-Quickstart#use-sdk-automation-from-rest-api-specifications) in TypeSpec Java QuickStart.
 
-- `<ServiceName>`: The RP name, which is usually same as rp name in swagger.
-- `<PackageName>`: Java package name.
-- `<Namespace>`: It should start with `com.`, and the remaining part can be got from `<PackageName>` by replacing `- with `.'. For example, if `<PackageName>` is `azure-analytics-purview-administration`, then `<Namespace>` is `com.azure.analytics.purview.administration`.
+`flavor` is always `azure` for Azure SDK.
+`examples-directory` is same as that in e.g. typespec-autorest emitter.
```

Comment:
Do we need to mention `service-dir`? Though it's cross language..

Reply:
Yeah, that's the reason I didn't mention this. I assume dev should get it from https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md#service-folder-structure
Emm, however that doc not explicitly talk about `service-dir`.

Anyway, we can add this later, if we see more misconfigure on that.

Reply:
We've mentioned it in our QuickStart https://github.com/Azure/azure-sdk-for-java/wiki/TypeSpec-Java-Quickstart#configuration